test/js/explore/explore-display-tabs.test.js

```diff
@@ -18,6 +18,16 @@ jest.mock('components/visualization/InferCNVIdeogram', () => {
 })
 
 import { getEnabledTabs } from 'components/explore/ExploreDisplayTabs'
+import React from 'react'
+import { render, screen, waitForElementToBeRemoved } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+import * as Reach from '@reach/router'
+import jquery from 'jquery'
+
+import MockRouter from '../lib/MockRouter'
+import * as ScpApi from 'lib/scp-api'
+import ExploreView from 'components/explore/ExploreView'
+import { mockPerformance } from '../mock-performance'
```

Comment:
BLOCKER: we can remove all these imports

Reply:
Ironically, I [suggested](https://github.com/broadinstitute/single_cell_portal_core/pull/1525#discussion_r892394449) the same.  Removed!